process_nwb/wavelet_transform.py

```diff
@@ -60,6 +61,39 @@ def hamming(n_time, rate, min_freq, max_freq):
 
     return k
 
+def get_filterbank(filters, n_time, rate, hg_only):
```

Comment:
You'll need to rebase and pull in the changes around nyquist check to this function.